modules/apps/29-fee/types/fee_test.go

```diff
@@ -198,31 +201,33 @@ func TestPacketFeeValidation(t *testing.T) {
 				packetFee.Fee.RecvFee = sdk.Coins{}
 				packetFee.Fee.TimeoutFee = sdk.Coins{}
 			},
-			false,
+			ibcerrors.ErrInvalidCoins,
 		},
 		{
 			"should fail with non empty Relayers",
 			func() {
 				packetFee.Relayers = []string{"relayer"}
 			},
-			false,
+			types.ErrRelayersNotEmpty,
 		},
 	}
 
 	for _, tc := range testCases {
 		tc := tc
 
-		fee := types.NewFee(defaultRecvFee, defaultAckFee, defaultTimeoutFee)
-		packetFee = types.NewPacketFee(fee, defaultAccAddress, nil)
+		t.Run(tc.name, func(t *testing.T) {
+			fee := types.NewFee(defaultRecvFee, defaultAckFee, defaultTimeoutFee)
+			packetFee = types.NewPacketFee(fee, defaultAccAddress, nil)
 
-		tc.malleate() // malleate mutates test data
+			tc.malleate() // malleate mutates test data
 
-		err := packetFee.Validate()
+			err := packetFee.Validate()
 
-		if tc.expPass {
-			require.NoError(t, err, tc.name)
-		} else {
-			require.Error(t, err, tc.name)
-		}
+			if tc.expErr == nil {
+				require.NoError(t, err, tc.name)
+			} else {
+				require.True(t, errors.Is(err, tc.expErr) || strings.Contains(err.Error(), tc.expErr.Error()), err.Error())
+			}
```

Comment:
I wonder would it be worth while turning this into some helper within the tests, seems like we need it in a good few places where errors aren't wrapped.

Reply:
yeah makes sense, done!